File: tests/files/extract_var_works_before.cs
class Program
{
    static void Main(string[] args)
    {
        Console.WriteLine("foo");

        while (false)
        {
            Console.WriteLine("foo");
        }

        do
        {
            Console.WriteLine("foo");
        } while (false);

        if (true)
        {
            Console.WriteLine("foo");
        }
        else
        {
            Console.WriteLine("foo");
        }

        for (int i = 0; i < 5; i++)
        {
            Console.WriteLine("foo");
        }
    }
}

File: tests/files/extract_var_works_before.php
<?php

function foo()
{
    print ('foo');

    while (true) {
        print ('foo');
        break;
    }

    do {
        print ('foo');
    } while (false);

    if (true) {
        print ('foo');
    } else {
        print ('foo');
    }

    for ($i = 0; $i < 5; $i++) {
        print ('foo');
    }
}
